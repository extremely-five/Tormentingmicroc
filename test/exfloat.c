void main(){
    float a ;
    float b;
    a=3.0;
    b=2.0;
    print(a*b);
}