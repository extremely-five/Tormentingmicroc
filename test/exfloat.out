24 19 0 5 25 15 1 15 1 13 26 1077936128 12 15 -1 13 0 1 1 26 1073741824 12 15 -1 13 11 13 0 1 1 11 3 22 21 2